eth/contracts/Most.sol

```diff
@@ -115,6 +117,35 @@ contract Most is Initializable, UUPSUpgradeable, Ownable2StepUpgradeable {
         requestNonce++;
     }
 
+    // Invoke this tx to transfer funds to the destination chain.
+    // Account needs to send native ETH which are wrapped to wETH
+    // tokens.
+    //
+    // Tx emits a CrosschainTransferRequest event that the relayers listen to
+    // & forward to the destination chain.
+    function sendRequestNative(
+        bytes32 destReceiverAddress
+    ) external payable {
+        uint256 amount = msg.value;
+
+        // check if the token is supported
+        bytes32 destTokenAddress = supportedPairs[addressToBytes32(wethAddress)];
+        require(destTokenAddress != 0x0, "Unsupported pair");
+
+        (bool success, ) = wethAddress.call{value: amount}(abi.encodeWithSignature("deposit()"));
```

Comment:
Since `wethAddress` is `payable` you should be able to use `weth.send` here. (calling `deposit` directly should not be required, but you can check)